docs: change terminology (#347)
askui/askui: docs/docs/general/08-Terminology.md

| Term                   | Meaning     |
| ---------------------- | ----------- |
| access token           | Gives you access to the askui inference in combination with your `workspace id`. Every `access token` has an expiry date and is assigned to exactly one workspace.  |
| action                 |A method in the askui Control Client API that describes an action to be taken against the operating system. For example click(), type(). |
| annotation             | Marked area around an element with metadata name, text, and bounding box coordinates. |
| automation             | A system of multiple connected workflows. |
| bounding box           | Rectangle described by coordinates that describe an element's location. Displayed visually as red rectangle. |
| credentials            | Combination of `workspace id` and `access token` living in the same workspace. |
| element                | A user interface component which websites and blocks are build from. |

> | element-description    | A method in the askui Control Client API that searches for a specific type of UI element. For example button(), textfield(). |

Reviewer: ```suggestion
| element-description    | A description for a UI element. For example in the askui Control Client API it is a coded description like `button()` or `textfield().contains().text().withText('Email')`. |
```
